src/common/Button/tests/Button.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import Button from '../Button';
+
+describe('Button component', () => {
+	it('renders the Button component with label', () => {
```

Comment:
There is a convention about the naming of test cases, and each test case should start with 'should' word. 
`it('should render the button with label')`